aptos-move/aptos-vm/src/move_vm_ext/vm.rs

```diff
@@ -61,6 +63,8 @@ impl MoveVmExt {
 
         extensions.add(NativeTableContext::new(txn_hash, remote));
         extensions.add(NativeRistrettoPointContext::new());
+        extensions.add(BellmanContext::new());
+        extensions.add(Bls12381Context::new());
```

Comment:
So this will require a different context for each elliptic curve we are implementing. I don't see a problem for now. Plus, I think folks will let us know if `extensions` gets too large.